### PR TITLE
Docs: Update README

### PR DESCRIPTION
**Pull request by Olexandr88**

### CODE REVIEW

**Comment by malb** [comment]
Thanks.

However,
- at least some of the links are clearly wrong, see https://github.com/malb/lattice-estimator/graphs/contributors for a list of contributors,
- a PR should have a description of what it's trying to achieve.

**Comment by Olexandr88** [comment]
Ok

**Comment by Olexandr88** [comment]
I corrected what I found, please check again

File: README.rst [hunk 1]
```diff
@@ -139,21 +139,21 @@ Contributions
...
-- Rachel Player
-- Sam Scott
+- [Hamish Hunt](https://github.com/hamishun)
+- [Hunter Kippen](https://github.com/hunterkipt)
+- [James Owen](https://github.com/jo276)
+- [Léo Ducas](https://github.com/lducas)
+- [Ludo Pulles](https://github.com/ludopulles)
+- [Markus Schmidt](https://github.com/swissiety)
+- [Martin Albrecht](https://github.com/malb)
+- [Michael Walter](https://github.com/catch22)
```
Comment (malb) [inline README.rst:154]: This isn't correct.

File: README.rst [hunk 2]
```diff
@@ -139,21 +139,21 @@ Contributions
...
-- Martin Albrecht
-- Michael Walter
-- Rachel Player
-- Sam Scott
+- [Hamish Hunt](https://github.com/hamishun)
+- [Hunter Kippen](https://github.com/hunterkipt)
+- [James Owen](https://github.com/jo276)
+- [Léo Ducas](https://github.com/lducas)
+- [Ludo Pulles](https://github.com/ludopulles)
+- [Markus Schmidt](https://github.com/swissiety)
```
Comment (malb) [inline README.rst:152]: Not sure this is correct.

File: README.rst [hunk 3]
```diff
@@ -139,21 +139,21 @@ Contributions
...
-- Léo Ducas
-- Ludo Pulles
-- Markus Schmidt
-- Martin Albrecht
-- Michael Walter
-- Rachel Player
-- Sam Scott
+- [Hamish Hunt](https://github.com/hamishun)
+- [Hunter Kippen](https://github.com/hunterkipt)
+- [James Owen](https://github.com/jo276)
```
Comment (malb) [inline README.rst:149]: This doesn't look correct.

**Pull request closed without merging**